HTML/QuickForm/hiddenselect.php

```diff
@@ -105,10 +105,14 @@ function toHtml()
     // }}}
     // {{{ accept()
 
-   /**
-    * This is essentially a hidden element and should be rendered as one
-    */
-    function accept(&$renderer)
+    /**
```

Comment:
change 9

Reply:
parent is ```
    function accept(&$renderer, $required=false, $error=null)
    {
        $renderer->renderElement($this, $required, $error);
    } // end func accept

    // }}}
```

Reply:
https://gist.github.com/seamuslee001/0a67ad08359f46b183a7e26f05c99658#file-quickform_diff-diff-L851 change comes directly from upstream